srcpkgs/python3-jupyter_nbconvert/template

```diff
@@ -1,24 +1,26 @@
 # Template file for 'python3-jupyter_nbconvert'
 pkgname=python3-jupyter_nbconvert
-version=6.0.7
-revision=2
+version=6.4.5
+revision=1
 wrksrc="nbconvert-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-mistune python3-Jinja2 python3-jupyterlab_pygments
  python3-nbclient python3-jupyter_core python3-entrypoints python3-testpath
- python3-bleach python3-pandocfilters python3-defusedxml"
+ python3-bleach python3-pandocfilters python3-defusedxml
+ python3-BeautifulSoup4"
 checkdepends="python3-pytest python3-ipython_ipykernel
  python3-jupyter_ipywidgets python3-Pebble pandoc inkscape $depends"
 short_desc="Converting Jupyter notebooks"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
 license="BSD-3-Clause"
 homepage="https://github.com/jupyter/nbconvert"
 distfiles="${PYPI_SITE}/n/nbconvert/nbconvert-${version}.tar.gz"
-checksum=cbbc13a86dfbd4d1b5dee106539de0795b4db156c894c2c5dc382062bbc29002
+checksum=21163a8e2073c07109ca8f398836e45efdba2aacea68d6f75a8a545fef070d4e
 conflicts="python-jupyter_nbconvert<=5.4.1_2"
 
 do_check() {
+	export PYTHONPATH=$(cd build/lib* && pwd)
```

Comment:
This is par for the course. In fact, the default `do_check` implementation does this automatically. You might even be able to get away with removing `do_check` altogether and setting
```sh
make_check_args="-k not((TestWebPDFExporter)or(test_webpdf_with_chromium))"
```
See, *e.g.*, `python3-dotenv`. (The unnatural syntax and lack of spacing is required since any attempt to quote spaces in `make_check_args` will fail.)

Reply:
That is ugly. Also, `-k` doesn't seem to support syntax like `-k not(TestWebPDFExporter::test_webpdf_without_chromium)`. One can get away with `--deselect` but this requires passing the complete prefix (which includes the full path to the python file where the test is located).

I came up with this
```
make_check_args="-m not(network) -k not(test_webpdf_without_chromium)"
```
matching precisely the three tests that have to be disabled... but it's hard to extend, I'm almost inclined to prefer the completely explicit (but very verbose)
```
make_check_args="
 --deselect nbconvert/exporters/tests/test_webpdf.py::TestWebPDFExporter::test_export
 --deselect nbconvert/exporters/tests/test_webpdf.py::TestWebPDFExporter::test_webpdf_without_chromium
 --deselect nbconvert/tests/test_nbconvertapp.py::TestNbConvertApp::test_webpdf_with_chromium"
```
At least with this syntax is very clear what tests are being deselected and how to add more tests later...

I guess the optimal situation would be if upstream would mark these tests as requiring pyppeteer and skiping them if not available.

Reply:
I don't have a strong preference; you could have even left `do_check` defined if it's easier and cleaner. The version you've pushed now seems fine.